scripts/health/common.py

```diff
@@ -0,0 +1,40 @@
+""" """
+import pandas as pd
+import requests
+
+
+def query_who(code: str):
+    """Query the WHO website for a given code.
+    To be replaced in bblocks
+    """
+
+    URL = 'https://ghoapi.azureedge.net/api/'
+
+    request = requests.get(URL + code)
+    data = request.json()
+    df = pd.DataFrame.from_records(data['value'])
+
+    return df
+
+
+def get_malaria_data() -> dict:
+    """Extract and clean malaria data for overview chart"""
+
+    malaria_dict = (query_who('MALARIA_EST_DEATHS')
+        .loc[lambda d: d.SpatialDim.isin(['GLOBAL', 'AFR']), ['SpatialDim', 'TimeDim', 'NumericValue']]
+        .sort_values('TimeDim')
+        .groupby(['SpatialDim'], as_index=False)
+        .last()
+        .assign(NumericValue=lambda d: pd.to_numeric(d.NumericValue))
```

Comment:
When you use `pd.to_numeric` please be explicit about what you want to happen to errors. Helps with readability